source/System.cpp

```diff
@@ -721,29 +721,29 @@ double System::RamscoopFuel(double shipRamscoop, double scale) const
 // Additional travel distance to target for ships entering through hyperspace.
 double System::ExtraHyperArrivalDistance() const
 {
-	return extraHyperArrivalDistance;
+	return max(extraHyperArrivalDistance, GameData::GetGamerules().SystemArrivalMin());
```

Comment:
What if we have a situation where we *want* the player to arrive closer to the system centre than usual? Or we have a system were we want to have a low departure distance, because of hazards near the system edge?